src/main/webapp/user-page.html

```diff
@@ -62,69 +62,54 @@ <h1 class="site-heading text-center text-white d-none d-lg-block">
       </div>
     </nav>
 
-    <!-- Text on background -->
-    <section class="page-section cta">
+
+    <!-- Message display -->
+    <section class="page-section cta" id="messages">
+
+      <!-- Text on background -->
       <div class="container">
         <div class="row">
           <div class="col-xl-9 mx-auto">
             <div class="cta-inner text-center rounded">
               <h2 class="section-heading mb-4">
-                <span class="section-heading-lower">Write a new post:</span>
+                <span class="section-heading-upper">Find more ...</span>
```

Comment:
Nit:
```suggestion
                <span class="section-heading-upper">Find out more</span>
```
Or maybe just remove this line